sessions/react-global-state/lights/pages/index.js

```diff
@@ -6,7 +6,7 @@ export default function HomePage() {
       <h1>Home</h1>
       <p>?? light(s) are on.</p>
       <p>
-        <Link href="/lights">All lights →</Link>
+        <Link href="/rooms">All rooms →</Link>
```

Comment:
not really an issue but I would keep it consistent with the "/lights" page:
```suggestion
        <Link href="/rooms">All Rooms →</Link>
```

Reply:
Thanks, done :)